internal/pkg/service/appsproxy/proxy/pagewriter/template/error.gohtml

```diff
@@ -110,9 +110,8 @@
         <path d="M20.5714 14.5714C20.5714 13.1518 21.7232 12 23.1429 12H24.8571C26.2768 12 27.4286 13.1518 27.4286 14.5714V33.4286C27.4286 34.8482 26.2768 36 24.8571 36H23.1429C21.7232 36 20.5714 34.8482 20.5714 33.4286V14.5714ZM12 24.8571C12 23.4375 13.1518 22.2857 14.5714 22.2857H16.2857C17.7054 22.2857 18.8571 23.4375 18.8571 24.8571V33.4286C18.8571 34.8482 17.7054 36 16.2857 36H14.5714C13.1518 36 12 34.8482 12 33.4286V24.8571ZM31.7143 15.4286H33.4286C34.8482 15.4286 36 16.5804 36 18V33.4286C36 34.8482 34.8482 36 33.4286 36H31.7143C30.2946 36 29.1429 34.8482 29.1429 33.4286V18C29.1429 16.5804 30.2946 15.4286 31.7143 15.4286Z" fill="white"/>
     </svg>
     {{ if .App }}
-       <h1>
-           {{.App.IDAndName}}
-       </h1>
+      <h1>Application {{.App.IDAndName}}</h1>
+      {{ if .App.ProjectID }} <h2>Project {{.App.ProjectID}}</h2>{{ end }}
     {{ end }}
 </header>
 <main>
```

Comment:
Small improvements in templates, so it is temporary better, until the design will be fixed:

![image](https://github.com/keboola/keboola-as-code/assets/19371734/8893097a-1fe5-4864-b588-a2cbc0414eed)

![image](https://github.com/keboola/keboola-as-code/assets/19371734/068516bb-50e5-47c5-9da1-7bf36805ef95)